test/surface/run-solid-test-suite.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -i
+npm start &
```

Comment:
Shall we just store the process ID and ensure the server is stopped?
As in https://github.com/solid/community-server/blob/dc8477c3f9507812b640dfcc0111d45f22d4b061/.travis/validate-package.sh#L11

Reply:
ah nice

Reply:
done